### ISSUE TITLE
Fuckr is a failed project

### ISSUE THREAD

**Issue by Grindr13** [tomlandia/fuckr#180]
Fuckr has been down since July 6, 2017 (over 7 months). There are 49 open issues that will never get resolved. You people are fucking retarded. Just give it up. Get rid of Fuckr altogether. What a waste.

**Comment by remydu34** [comment]
yeah, i have the same problem as you boy,... fuckr tell me HTTP 412 .... its annoying ;(

**Comment by AlahmadiQ8** [comment]
your comments are disrespectful and only doing harm to the open source community. [The Open Source Community Has No Place for Disrespect](https://medium.com/@d4nyll/the-open-source-community-have-no-place-for-disrespect-70c85d473332)

**Comment by RobbieTechie** [comment]
@remydu34 the 412 issue was already dealt with and is covered in other posts

**Comment by remydu34** [comment]
Yeah I know ;) but where is the solution ;( . Your fuckr application works
in your computer ?

Le vendredi 9 février 2018, RobbieTechie <notifications@github.com> a
écrit :

**Comment by RobbieTechie** [comment]
Are you on a Windows computer.  I don't have a Mac so I can't open and edit the Mac version.  The file you need to replace that is causing the 412 is package.json I'm attaching it here, it's not actually a zip file, so just remove the .zip extension so it's back to package.json then copy it into the main folder for Fuckr
[package.json.zip](https://github.com/tomlandia/fuckr/files/1713982/package.json.zip)

**Comment by remydu34** [comment]
Ok I'm in my bed , I will try ahh I will tell you if it's works :) thanks
so much

Le dimanche 11 février 2018, RobbieTechie <notifications@github.com> a
écrit :

**Comment by remydu34** [comment]
can you send me some screen-shoot im not so stronger with PC ahah ? because
i don"t which .zip i have to remove and where i have to put (
package.json.zip) , so maybe you can hep me with images, or videos just to
show me what should I do :) i have windows 7 . Merci beaucoup

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2018-02-11 20:27 GMT+01:00 REMY Tourrette <remy.tourrette@gmail.com>:

**Comment by remydu34** [comment]
hm ;( your zip package.json.zip its empty .. it is normal ?

2018-02-11 20:34 GMT+01:00 REMY Tourrette <remy.tourrette@gmail.com>:

**Comment by xfy741** [comment]
Thank you RobbieTechie, I finally get fuckr to work. It's what i was looking for, a map of users, I was trilateration but that was time consuming. Thank you  again for your help on this project!

**Comment by remydu34** [comment]
PLEASE XFY741 help me :( please explain me ;)

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2018-02-13 6:30 GMT+01:00 xfy741 <notifications@github.com>:

**Comment by xfy741** [comment]
Remydu34 I just downloaded the program https://github.com/tomlandia/fuckr/releases/download/2.0.0-alpha/Fuckr-win64.zip and extracted the zip to a folder, then I donwloaded the file provided by RobbieTechie https://github.com/tomlandia/fuckr/files/1713982/package.json.zip but I didn´t extracted it, I just eliminate the .zip extension, then I copy package.json to the extracted fuckr folder and for me it worked!

**Comment by JohnnyAccorn** [comment]
So if there is a working solution for Windoze why can't the website download packake be updated so we can all enjoy a working copy.
One solution I'm working on is an app working alongside grindr using its database, the app just displays markers of users like fuckr. Still working on it but would like a working version (windows) if at all possible.

**Comment by RobbieTechie** [comment]
That would be up to @tomlandia who is the author of Fuckr.  This is his github so he would have to update the packages or close issues, or any parts that arise here on github.

**Comment by remydu34** [comment]
HELLO xfy741 :)

thanks to replied :) But where is this, so ok i have fuckr-win64.zip, then
i extracted that's ok, then i donwloaded package.json.zip that's ok too ..
but where is " ZIP.EXTENSION " ? that i have eliminate ? and where exactly
I have to copy package.json.zip ? thanks ;) maybe do you have snaptacht or
somethings to explain me with videos ?

Thank you so much :)

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2018-02-18 0:58 GMT+01:00 JohnnyAccorn <notifications@github.com>:

**Comment by remydu34** [comment]
hello Johnny :) that's cool :) hope you will find a solution :) !


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2018-02-18 0:58 GMT+01:00 JohnnyAccorn <notifications@github.com>:

**Comment by xfy741** [comment]
If you are on windows you have to set the file explorer to unhide and see the common file extension, here you can find a guide to do that: http://kb.winzip.com/kb/entry/26/ then you will see the file package.json.zip like this. You have to right click on it and choose Rename (F2 works also) another guide: http://www.mediacollege.com/microsoft/windows/extension-change.html, then you have to place your cursor to the end of the name and backspace, delete the .zip extension, confirm yes and then the file become package.json, copy this file to the extracted fuckr folder, you don't need to place it in another folder just the main fuckr folder.

**Comment by xfy741** [comment]
Hi JohnnyAccorn, I don't know programming but I am very curious about computers and software. I use Koplayer and Nox emulator for grindr, I extracted the app data and it's easy to get all the pictures but they're without the messages so it's really hard to organize that stuff. The chat table .DB has all the information and I saw it with a DB viewer. Is there a way to organize all that info so you can get the pictures that belong to each user? I would love to have my messages like fuckr and just drag and drop the pictures in a folder for each user I have talked with. I would save a lot of time.

**Comment by remydu34** [comment]
OMG its working now ! thanks so much ! but now i have another problem .. i
can't see the map but its ok lol .

THANK YOU ! Merci ! :)

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2018-02-19 17:54 GMT+01:00 xfy741 <notifications@github.com>:

**Comment by grindr14** [comment]
@AlahmadiQ8 of course you would make a comment like that. This is not even about race. This is about incompetent people who don't know how to manage an app. But since you brought up race, why don't you go back to the Middle East where you belong? Get out of my country you fucking terrorist!

**Comment by cjiom** [comment]
Just updated my package.json to get in

  "user-agent": "Grindr3/33809 (iPhone; iOS 11.0)",

so looks like they have blocked some older versions again.

**Comment by remydu34** [comment]
Hello guys ;)

Help me guys , I have a problem with Fuckr again ... so it's with (
Unexpected error HTTP 404.) if someone know a solution ... ;) Thank you !
Merci !

**Comment by dax702** [comment]
I get unexpected error (HTTP 412) when I try to login or create an account, using windows 64. Anyone else?

**Comment by cjiom** [comment]
Rename your fuckr.js file to fuckr.bak
copy in fuckr.txt below, then rename fuckr.js (you'll need the option selected in Folder Options to show file extensions to do this).

[fuckr.txt](https://github.com/tomlandia/fuckr/files/2002929/fuckr.txt)

Do the same for Package.json, rename existing replace with package.txt and rename etc. 
[package.txt](https://github.com/tomlandia/fuckr/files/2002931/package.txt)

These work for me.

**Comment by dax702** [comment]
Thanks, it's sort of working now
